src/modules/02.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Die Nutzer:innenperspektive
+title: Die Nutzer:innen&shy;perspektive
```

Comment:
I introduces this in the first place but I had to revert this because it is included in the slug. :(

Reply:
Added a filter that removes the `&shy;` for slugs! ([8d5d620](https://github.com/technologiestiftung/service-agentinnen/pull/7/commits/8d5d620dd4947887f52d4244884db1d48938fd6c))

Reply:
Great! Well done!